pages/en/contrib/code-contrib-lb2-lb3.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Contributing code in LoopBack 2.x and 3.x
+lang: en
+tags: [community, contributing, getting_started]
+keywords: LoopBack, open-source
+summary: "This article explains the process and guidelines for contributing code to the LoopBack 2.x and 3.x projects."
+sidebar: contrib_sidebar
+permalink: /doc/en/contrib/code-contrib-lb2-lb3.html
+---
+
+
+## How to contribute to the code
+
+You can contribute in a number of ways:
+
+* [Report an issue](#reporting-an-issue).   Reporting it first will ensure that it is
+  in fact an issue, and there is not already a fix in the works.
+* After reporting a bug, go one step further and fix it.
+* Ensure that your effort is aligned with the project's roadmap by
+  talking to the maintainers, especially if you are going to spend a
+  lot of time on it.
+* Adhere to code style outlined in the [style guide](style-guide.html).
+* Agree to the [Contributor License Agreement](#agreeing-to-the-cla).
+* [Submit a pull request](#submitting-a-pull-request) through Github.
+* Run tests; see [Running tests](#running-tests).
+
+## Where to start
+
+If you are looking to contribute, but aren't sure where to start, review open pull requests for the module.  For example: [LoopBack open pull requests](https://github.com/strongloop/loopback/pulls).
+
+Also, please read the source code before contributing. If something is confusing, poorly documented, poorly commented, or could otherwise be improved: open a GitHub issue and link to the file and line of code with a useful description of what you would like to see or think is missing.
+
+Another place to look is to find our issues on GitHub with `good first issue` tag.  See 
+[good first issue in our repositories](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22good+first+issue%22+user%3Astrongloop).
+
+### LoopBack versions
+
+LoopBack maintains a current version, a long-term support (LTS) version, and a maintenance version.  New features go into the current version; bug fixes can go into the LTS and maintenance versions, depending on the nature of the bug.  For more information, see [Long-term support](Long-term-support.html).
+
+### Recommended Node.js version
+
+{% include note.html content="Check your Node.js version with the command `node -v`.
+" %}
+
+LoopBack 2.x supports Node.js v0.10 and v0.12, but LoopBack 3.0 does not support Node.js v0.10 or v0.12.  
```

Comment:
This is no longer accurate, nobody should be worrying about pre-4.0 versions of Node.js anymore. (Node.js 4.x is not supported anymore either.)

Since not all of LB3 modules support Node.js 10 LTS (from what I vaguely remember), I am proposing to recommend Node.js 8 as the version to use.


E.g.

```markdown
We recommend our contributors to use Node.js 8 for developing and testing LoopBack changes.
```